src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,37 @@
+import { Link } from "react-router-dom";
+import React, { useState, useEffect } from 'react'
+import { useParams } from "react-router-dom";
+
```

Comment:
I'll leave this file for now as it's not related to the current ticket. Please remove and include this in the next task so we are keeping to the one task at a time convention and committing meaningful code 👍